packages/react/src/primitives/Field/FieldDescription.tsx

```diff
@@ -8,12 +9,15 @@ export const FieldDescription: React.FC<FieldDescriptionProps> = ({
   descriptiveText,
   labelHidden,
   ...rest
-}) => {
-  return !labelHidden && descriptiveText ? (
-    <Text className={ComponentClassNames.FieldDescription} {...rest}>
-      {descriptiveText}
-    </Text>
-  ) : null;
-};
+}) => (
+  <Text
+    className={classNames(ComponentClassNames.FieldDescription, {
+      [ComponentClassNames.VisuallyHidden]: labelHidden,
```

Comment:
Need to also update the conditional in each field component that uses FieldDescription.
```
const ariaDescribedBy = descriptiveText ? descriptionId : undefined
```

This will ensure we only add the aria-describedby property when customers actually provide descriptive text. Otherwise we are linking to an empty div.

Reply:
if `ariaDescribedBy` is `undefined`, we would get ADA error?

Reply:
If ariaDescribedBy is undefined, then it shouldn't add the prop to the dom, which should avoid the accessibility error. If there's no descriptiveText provided by the user, then it doesn't make sense to have an ariaDescribedBy at all (thus why I think we should set it to undefined). Does that make sense? Happy to discuss offline if that's easier.

Reply:
That makes sense! I updated the PR and also made the same changes for all the `aria-describedby` in the code base 😁